handlers/meshery_filter_handler.go

```diff
@@ -21,8 +21,8 @@ type MesheryFilterRequestBody struct {
 // swagger:route GET /api/experimental/filter/file/{id} FiltersAPI idGetFilterFiles
 // Handle GET request for filter file with given id
 //
-// Returns the Meshery Filter file saved by the current user with the given id
-// This will return the Meshery Filter file with their details of the given id
+// Returns the Meshery Filters file saved by the current user with the given id
+// This will return the Meshery Filters file with their details of the given id
```

Comment:
comment repeated x2

Reply:
This should be changed to Filter right ?